docs/source/api.rst

```diff
@@ -26,7 +26,7 @@ These more general Dask functions are described below:
 These functions work with any scheduler.  More advanced operations are
 available when using the newer scheduler and starting a
 :obj:`dask.distributed.Client` (which, despite its name, runs nicely on a
-single machine).  This API provides the ability to submit, cancel, and track
+single machine.)  This API provides the ability to submit, cancel, and track
```

Comment:
I thought that the convention here was to place the period after the parenthetical.  My understanding is that you should be able to remove the parenthetical entirely and that the syntax should still be valid.

Reply:
My mistake, apologies. I'll revert this change and add this rule to my notes to avoid doing it again.